src/renderer/components/app.tsx

```diff
@@ -107,8 +107,22 @@ export class App extends React.Component {
   }
 
   componentDidMount() {
+    const stores: Array<KubeObjectStore> = [namespaceStore];
```

Comment:
`namespaceStore` has logic not to start watching if there are no accessible namespaces available, but should we still have check here, if user has not configured any namespaces in cluster settings?

Reply:
I think we need to fix this within `KubeObjectStore`, not in each react component.